lib/tropic/tropic_paragraph.flow

```diff
@@ -210,6 +220,11 @@ spaceStr : string = " ";
 zeroSpaceStr : string = "\u200b";
 
 TParaLine(words : [TParaWord], indent : double);
+WordsAndIndexes(words : [TParaWord], indexes : [int]);
+
+makeRest(array : [?], count : int) -> [?] {
+	subrange(array, 1, length(array) - count);
```

Comment:
looking here https://github.com/area9innovation/flow9/pull/545/files#diff-9b8fade5cbf279ff9a19f9bbc55111a7L360
makeRest should be subrange(array, count, length(array) - count); and this is taleFrom

Reply:
> looking here https://github.com/area9innovation/flow9/pull/545/files#diff-9b8fade5cbf279ff9a19f9bbc55111a7L360
> makeRest should be subrange(array, count, length(array) - count); and this is taleFrom

Yes, my bad. Thank you